Safety/boardfiles/Src/main.cpp

```diff
@@ -77,6 +78,7 @@
 
 /* Private function prototypes -----------------------------------------------*/
 void SystemClock_Config(void);
+void toggleLED();
```

Comment:
This file doesn't look like it's meant for review yet, so I'll leave it un-reviewed.

Reply:
Ya main.cpp is for testing rn lol